### PR TITLE
Update get-file-statistics.md

### PR DESCRIPTION
**Pull request by msbemba**
Added max lookbackhours

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5886

File: microsoft-365/security/defender-endpoint/get-file-statistics.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Retrieves the statistics for the given file.
 ## Limitations
 
 1. Rate limitations for this API are 100 calls per minute and 1500 calls per hour.
+2. Maximum Value for Lookbackhours is 720 Hours(30days).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-file-statistics.md]:
```suggestion
2. Maximum value for Lookback hours is 720 hours(30 days).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested change. Many thanks.

File: microsoft-365/security/defender-endpoint/get-file-statistics.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/security/defender-endpoint/get-file-statistics.md]: lookBackHours is the exact match of the query. So this change is not needed.

Comment (JohanFreelancer9) [reply microsoft-365/security/defender-endpoint/get-file-statistics.md]: @msbemba, thank you for clarifying.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**